Modules/Shipping/Http/Services/ShippingService.php

```diff
@@ -43,7 +38,7 @@ public function create(array $requestData): array
      */
     public function createShipping(array $shippingData): Shipping
     {
-
+        $shippingData['user_id'] = auth()->user()->id;
         $shipping = new Shipping();
         $shipping->fill($shippingData);
```

Comment:
```$shipping->fill($shippingData);``` will fall if front end will send same id 2 times, so delete id before filling data

Reply:
From fronted there is no user_id parameter pass to api